tests/Oryx.BuildImage.Tests/Python/PythonSampleAppsTest.cs

```diff
@@ -573,6 +573,7 @@ public void ErrorDuringBuild_ResultsIn_NonSuccessfulExitCode()
                 $"{appDir} --platform {PythonConstants.PlatformName} " +
                 $"--platform-version {langVersion} > {generatedScript}")
                 .SetExecutePermissionOnFile(generatedScript)
+                .AddDefaultTestEnvironmentVariables()
```

Comment:
Looks like this is the only test failing Validation -- not sure if the line added here is causing it (though I do see Python 3.8.15 being used when the test states it uses 3.7, so this may be causing it), but it's worth checking locally to see if it reproduces.

Also it's not a bad idea to kick off a Nightly build against this branch to ensure that the additional tests are also successful.

Reply:
I think that this is failing since we are not setting the `AddDefaultTestEnvironmentVariables` before actually executing the oryx command, and so the environment variable isn't actually respected